packages/cli/test/acceptance/app-run.acceptance.js

```diff
@@ -14,25 +15,36 @@ const build = require('@loopback/build');
 describe('app-generator (SLOW)', function() {
   const generator = path.join(__dirname, '../../generators/app');
   const rootDir = path.join(__dirname, '../../../..');
-  const sandbox = path.join(rootDir, 'sandbox/sandbox-app');
+  const fullPathSandbox = path.join(rootDir, 'sandbox/sandbox-app');
+  const homeDir = process.env.HOME || process.env.userprofile;
+  const tildeSandbox = fullPathSandbox.replace(homeDir, '~');
```

Comment:
I'm not a fan of using a bunch of tiny dependencies.. but the `untildify` dependency being used was named as the opposite of its sister package, [`tildify`](https://www.npmjs.com/package/tildify), which does exactly what you want here.